lib/retry.js

```diff
@@ -4,37 +4,67 @@ const sleep = msec => new Promise(resolve => {
   setTimeout(resolve, msec);
 });
 
-const promisify = fn => (...args) => (
-  new Promise((resolve, reject) => (
-    fn(...args, (err, data) => (
-      err ? reject(err) : resolve(data)
-    ))
-  ))
-);
+const promisify = (fn, args) => new Promise((resolve, reject) => {
+  if (!args) args = [];
```

Comment:
```suggestion
const promisify = (fn, args = []) => new Promise((resolve, reject) => {
```
Maybe better to use default argument